### PR TITLE
compare timestamps of filename with timestamps from within nc-file

### PR DESCRIPTION
**Pull request by ulysses78**
If the timestamps from within the nc-file equals the timestamp of the filename, then the measurementtime from within the nc-file will be written to the mTime-variable. Otherwise the timestamp from the filename will be taken to fill the mTime-variable.

### CODE REVIEW

File: lib/io/readPollyRawData.m [hunk 1]
```diff
@@ -143,17 +145,30 @@
...
     [thisYear, thisMonth, thisDay, thisHour, thisMinute, thisSecond] = ...
-                       datevec(mTimeStart);
-    mTime(1, :) = thisYear * 1e4 + thisMonth * 1e2 + thisDay;
-    mTime(2, :) = thisHour * 3600 + ...
-                 thisMinute * 60 + ...
-                 thisSecond + 30 .* (0:(size(mTime, 2) - 1));
+                           datevec(mTimeStart);
+    mTime_file(1, :) = thisYear * 1e4 + thisMonth * 1e2 + thisDay;
+
+    if mTime_file(1, :) == mTime(1, :)
```
Comment (ZPYin) [inline lib/io/readPollyRawData.m:156]: I'm wondering what the value of `mTime(1, :)`. Did it equal to the start time of the measurement day? Or did it point to the time of the first profile in the measurement day?

For the latter scenario, this condition check does not confirm DAQ issue, if the first profile does not start from 00:00 UTC. 

What about 
```matlab
if mTime_file(1, :) == floor(mTime(1, :))
```

**Review by ZPYin — COMMENTED** [review]
Hi Andi,

It looks very nice. Just one little comment to consider. Please see my in-line comments.

Many thanks!

Zhenping

File: lib/io/readPollyRawData.m (hunk 1 above)
Comment (ulysses78) [reply lib/io/readPollyRawData.m:156]: The values of ```mTime(1, :)``` representing the date entries only (no seconds of the day) in the measurementtime variable from within the nc-file. This should always be the same date as is written in the filename of the nc-file. If not, something went wrong and mTime will be filled with the date from the filename. Hopefully there is no measurementfile which spans over two days (over 00:00:00 o'clock)!

Comment (ZPYin) [reply lib/io/readPollyRawData.m:156]: OK, got it.
Thanks for your explanation.

It's really to be merged.

**Pull request merged**